hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/ContextTransactionProcessor.java

```diff
@@ -129,7 +135,16 @@ public CallOutcome call() {
             }
             // Commit any HAPI fees that were charged before aborting
             rootProxyWorldUpdater.commit();
-            final var result = HederaEvmTransactionResult.fromAborted(e.senderId(), hevmTransaction, e.getStatus());
+
+            var result = HederaEvmTransactionResult.fromAborted(e.senderId(), hevmTransaction, e.getStatus());
+
+            if (context.body().hasEthereumTransaction()) {
+                final var sender = rootProxyWorldUpdater.getHederaAccount(e.senderId());
+                if (sender != null) {
```

Comment:
This check is required because it is possible that the sender does not exist. In this case a `INVALID_ACCOUNT_ID` error is thrown in `processor.processTransaction(...)` and it is handled in this `catch` statement. This is shown in the test `revertsWhenSenderDoesNotExist`